packages/shared/src/nomination-pools.ts

```diff
@@ -321,6 +368,30 @@ async function nominationPoolLifecycleTest(relayChain, addressEncoding: number)
 
   await check(nominationPoolWithCommission.commission).toMatchObject(newCommissionData)
 
+  /**
+   * Nominate a validator set
+   */
+
+  const validators = await getRandomValidators(relayClient.api, 100)
```

Comment:
any particular reason this is using random validators, not say first X validators? generally we want to avoid randomness in tests because that makes error reproduction hard

Reply:
I did this so that the test had another dimension along which it could vary.

This could be fixed by writing a seeded PRNG and fixing the seed (since `Math.random` can't be seeded), but your suggestion seems more reasonable.